### ISSUE TITLE
Edward Zhu's Evaluation *CRITICAL*

### ISSUE THREAD

**Issue by zhued** [Kevin-Rau/Breakfast#3]
### **breakfast.js**

_Line 3:_
skeleton.connect('mongodb://localhost/simple');
[Skeleton](https://www.npmjs.com/package/skeleton) is an Express framework-less app structure generator. It simply generates a skeleton of a basic structure.  The ".connect" method does not exist in the skeleton package. You're not using skeleton properly.  You need mongoose in order to call the connect method on a mongodb.  
### **DoStuff.js**

_Line 1:_
var database= require('./mongo.js');
There is no js file that configs a MongoDB service. How did you get this assignment actually working with that still in the code?

_Line 43:_
app.put('/person/:id', function(req, res){
Why did you decide on person/:id when your data in on breakfast foods and has nothing to do with people?

**Comment by kenbod** [comment]
Yes, it does appear that the assignment still needs some work to be functional. Would members of the team please respond to these comments and update the code so we have confidence your web service works?
